src/vos/tests/vts_mvcc.c

```diff
@@ -639,6 +708,10 @@ static struct op operations[] = {
 	{"queryc",	T_R,	L_C,	L_NIL,	R_R,	W_NIL,	NULL},
 	{"queryo",	T_R,	L_O,	L_NIL,	R_R,	W_NIL,	NULL},
 	{"queryd",	T_R,	L_D,	L_NIL,	R_R,	W_NIL,	NULL},
+	/* Fetch with updates */
```

Comment:
Comment should be "Read timestamp update only cases" or some such

Reply:
done